types/victory/index.d.ts

```diff
@@ -327,6 +327,112 @@ declare module 'victory' {
+         */
+        children?: React.ReactElement | React.ReactElement[];
+        /**
+         * The `circleComponent` specifies the element to use when a `VictoryClipContainer` renders
+         * a circular clip path. By default, `VictoryClipContainer` uses the `Circle` component.
+         * @default circleComponent={<Circle/>}
+         */
+        circleComponent?: React.ReactElement;
+        /**
+         * The `className` prop specifies a class name that will be applied to the rendered element.
+         * @example: className="myClipPath"
+         */
+        className?: string;
+        /**
+         * The `clipHeight` prop determines the base height of the rectangular clip path. This prop should be
+         * given as a number. If this prop is not given, it will be calculated based on the height and padding
+         * of the parent chart.
+         */
+        clipHeight?: number;
+        /**
+         * The `clipId` prop may be used to set a deterministic id for the container.
+         * When a `containerId` is not manually set, a unique id will be generated.
+         * It is usually necessary to set deterministic ids for automated testing.
+         */
+        clipId?: number | string;
+        /**
+         * The `clipPadding` prop is used when the clipped area should be larger than the range of a chart.
+         * This prop should be given as an object with `top`, `bottom`, `left`, and `right` properties. Set
+         * the `clipPadding` prop is useful for extending the visible area of a chart in some dimension so
+         * that data or labels are not cut off.
+         */
+        clipPadding?: {
+            top?: number;
+            bottom?: number;
+            left?: number;
+            right?: number;
+        };
+        /**
+         * The `clipPathComponent` prop specifies the clip path to apply to the rendered group when appropriate.
+         * By default, `VictoryClipContainer` uses the `ClipPath` component.
+         * @default clipPathComponent={<ClipPath/>}
+         */
+        clipPathComponent?: React.ReactElement;
+        /**
+         * The `clipWidth` prop determines the base width of the rectangular clip path. This prop should be
+         * given as a number. If this prop is not given, it will be calculated based on the width and padding
+         * of the parent chart.
+         */
+        clipWidth?: number;
+        /**
+         * The `events` prop attaches arbitrary event handlers to the group element.
+         * This prop should be given as an object of event names and corresponding event handlers.
+         * When events are provided via Victory's event system, event handlers will be called with
+         * the event, the props of the component it is attached to, and an `eventKey` when applicable.
+         * @example events={{onClick: (evt) => alert("x: " + evt.clientX)}}
+         */
+        events?: object;
```

Comment:
Maybe it's worth using the same type as [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e02df99913d0c992415eec15a2cb1ba33e8ccd6c/types/victory/index.d.ts#L181)?

Reply:
Great idea, didn't know about `DOMAttributes`, it's a really useful type.